packages/larva-patterns/objects/o-email-capture-form/o-email-capture-form.twig

```diff
@@ -5,7 +5,12 @@
 		{% include "@larva/components/c-button/c-button.twig" with c_button %}
 	</div>
 
-	<input type="hidden" name="__contextName" value="FormPost" />
+	{% if o_email_capture_form_context_name %}
```

Comment:
Don't need a condition here - in the o-email-capture-form.prototype you can add a key for  o_email_capture_form_context_name: 'FormPost' in the exported object, and when JSON is generated it will reference that.

Reply:
Could probably do the same for the execution context